### PR TITLE
10951 issuing distributor access tokens

### PR DESCRIPTION
**Pull request by ravi102465**
This PR has changes for Adding new get token endpoint in apim.
Changes on swagger document to add new endpoint
Changes on terraform scripts for policy
pipeline changes for passing parameter.
Added missing network restriction on log stash storage and explicit set allow_blob_public_access to false

### CODE REVIEW

**Comment by ravi102465** [comment]
Please hold the merge as there are some changes need to be added after today discussion

File: exchangeSetService_OpenApi_definition.yaml [hunk 1]
```diff
@@ -205,6 +205,70 @@ paths:
...
+        Returns a token direct from AAD using Client Credentials
+      requestBody:
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  client_id:
+                    type: string
+                    description: The application ID that's assigned to the distributor's app. This will be provided to the distributor when they register their application
```
Comment (martyn-fewtrell) [inline exchangeSetService_OpenApi_definition.yaml:222]: Really minor point but we should have a full stop at the end of the description.

**Review by martyn-fewtrell — COMMENTED** [review]
Just one very minor typo.

File: exchangeSetService_OpenApi_definition.yaml (hunk 1 above)
Comment (ravi102465) [reply exchangeSetService_OpenApi_definition.yaml]: fixed

**Pull request merged**